shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/statement/dml/InsertStatementContext.java

```diff
@@ -60,39 +64,45 @@ public final class InsertStatementContext extends CommonSQLStatementContext<Inse
     
     private final TablesContext tablesContext;
     
-    private final List<String> columnNames;
+    private final Map<Integer, List<String>> columnNames = new LinkedHashMap<>();
```

Comment:
If we only support insert into one table now, can we keep the InsertStatementContext class unchanged?

Reply:
OK, this will be implemented in federation module later.